src/groups/mwc/mwcu/mwcu_printutil.h

```diff
@@ -738,72 +748,71 @@ mwcu::operator<<(bsl::ostream&                      stream,
+mwcu::operator<<(bsl::ostream&                                   stream,
+                 const Printer<bsl::map<KEY, VAL, CMP, ALLOC> >& printer)
 {
     stream << "{";
 
     if (!printer.obj().empty()) {
-        typedef typename bsl::map<TYPE1, TYPE2, TYPE3>::const_iterator Iter;
+        typedef typename bsl::map<KEY, VAL, CMP, ALLOC>::const_iterator Iter;
         Iter iter     = printer.obj().begin();
         Iter lastElem = --printer.obj().end();
         for (; iter != lastElem; ++iter) {
-            stream << Printer<TYPE1>(&iter->first) << ":"
-                   << Printer<TYPE2>(&iter->second) << ", ";
+            stream << Printer<KEY>(&iter->first) << ":"
+                   << Printer<VAL>(&iter->second) << ", ";
         }
 
-        stream << Printer<TYPE1>(&iter->first) << ":"
-               << Printer<TYPE2>(&iter->second);
+        stream << Printer<KEY>(&iter->first) << ":"
+               << Printer<VAL>(&iter->second);
     }
 
     stream << "}";
 
     return stream;
 }
 
-template <typename TYPE1, typename TYPE2>
+template <typename KEY, typename CMP, typename ALLOC>
 inline bsl::ostream&
-mwcu::operator<<(bsl::ostream&                           stream,
-                 const Printer<bsl::set<TYPE1, TYPE2> >& printer)
+mwcu::operator<<(bsl::ostream&                              stream,
+                 const Printer<bsl::set<KEY, CMP, ALLOC> >& printer)
 {
     stream << "{";
 
     if (!printer.obj().empty()) {
-        typedef typename bsl::set<TYPE1, TYPE2>::const_iterator Iter;
+        typedef typename bsl::set<KEY, CMP, ALLOC>::const_iterator Iter;
         Iter iter     = printer.obj().begin();
         Iter lastElem = --printer.obj().end();
         for (; iter != lastElem; ++iter) {
-            stream << Printer<TYPE1>(&(*iter)) << ", ";
+            stream << Printer<KEY>(&(*iter)) << ", ";
         }
 
-        stream << Printer<TYPE1>(&(*iter));
+        stream << Printer<KEY>(&(*iter));
     }
 
     stream << "}";
 
     return stream;
 }
 
-template <typename TYPE1, typename TYPE2, typename TYPE3>
+template <typename KEY, typename HASH, typename KEY_EQ, typename ALLOC>
 inline bsl::ostream& mwcu::operator<<(
-    bsl::ostream&                                            stream,
-    const Printer<bsl::unordered_set<TYPE1, TYPE2, TYPE3> >& printer)
+    bsl::ostream&                                                 stream,
+    const Printer<bsl::unordered_set<KEY, HASH, KEY_EQ, ALLOC> >& printer)
 {
     stream << "{";
 
     if (!printer.obj().empty()) {
-        typedef
-            typename bsl::unordered_set<TYPE1, TYPE2, TYPE3>::const_iterator
-                Iter;
-        Iter    begin = printer.obj().begin();
-        Iter    end   = printer.obj().end();
+        typedef typename bsl::unordered_set<KEY, HASH, KEY_EQ, ALLOC>::
+            const_iterator Iter;
+        Iter               begin = printer.obj().begin();
+        Iter               end   = printer.obj().end();
```

Comment:
```suggestion
        const Iter         begin = printer.obj().begin();
        const Iter         end   = printer.obj().end();
```

Reply:
I would rather not introduce logic changes in this PR

Reply:
unless there is a reason to, like a compiler warning or something

Reply:
Let's merge it like this then